.github/ISSUE_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ questions regarding the behavior/reproduction for more than 20 days "dead". All
 
 ## Please, provide the details below:
 
-### Did you verify this is a real problem by searching the [NativeScript Forum](http://forum.nativescript.org) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?
+### Did you verify this is a real problem by searching [Stack Overflow](https://stackoverflow.com/questions/tagged/nativescript), the [NativeScript Forum](https://discourse.nativescript.org/), and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?
```

Comment:
@rdlauer why do we want to keep the discourse link here? Should I change it to something else ?

Reply:
Only because it's another source of resolution for folks...even if it'll be read-only. I'd be ok removing it though.